Source/Eto.Gtk/Forms/Controls/ScrollableHandler.cs

```diff
@@ -121,7 +120,8 @@ public override void AttachEvent(string id)
 			{
 				case Scrollable.ScrollEvent:
 					Control.Events |= Gdk.EventMask.ScrollMask;
-					Control.ScrollEvent += Connector.HandleScrollableScrollEvent;
+					Control.Vadjustment.ValueChanged += (sender, e) => Connector.HandleScrollableScrollEvent(ScrollPosition);
```

Comment:
I'm pretty sure this will still cause a memory leak as it creates a hard reference to the handler (similar to calling the Callback directly).  It should be something like this, changing the signature of `HandleScrollableScrollEvent` to match:
`Control.Vadjustment.ValueChanged += Connector.HandleScrollableScrollEvent;`